### PR TITLE
Add 1 branch and 1 index

### PR DESCRIPTION
**Pull request by KsushaDk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+     <title>My first page</title>
+     <meta charset="UTF-8"/>
+     <meta name="description" content="Homework "/>
+</head>
+<body>
+    <a href=file:///C:/Users/asus/Desktop/if-html/about/index.html>About author</a>
```
Comment (BubentsovKirill) [inline index.html:9]: Seems it should be like <a href="./about/index.html">About author</a>

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <p>As the class shook their heads in agreement, she continued, <em>“Your stresses and worries in life are very much like this glass of water.</em></p> 
+    <p><em>Think about them for a while and nothing happens.  Think about them a bit longer and you begin to ache a little.  Think about them all</em></p>  
+    <p><em>day long, and you will feel completely numb and paralyzed – incapable of doing anything else until you drop them.”</em></p> 
+    <br/>
+    <blockquote><strong> Moral of the Story:</strong>
+        <del>On days when you’re stressed out and feeling overwhelmed, learn to embrace the inevitable and let go. 
+        Some things cannot be controlled and no amount of worrying could put all your burdens away. Instead 
+        of letting stress get to you, embrace it and conquer it.</del> Let yesterday’s worries inspire you to a productive day.
+    </blockquote>
+    <hr>
```
Comment (BubentsovKirill) [inline about/index.html:34]: It should be closed.

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="UTF-8"/>
+    <meta name="description" content="Homework "/>
+    <title>About author</title>
+</head>
+<body>
+    <h1 style="background-color:aquamarine;">Hello.</h1>
+    <h2 style="background-color:blueviolet;">My name is Ksusha.</h2>
+    <h3 style="background-color:crimson;">I have problems with imagination so it will be a page with random text :)</h3>
+    <p style="color: darkblue;"><b>The Weight of the Glass.</b></p>
+    <hr>
```
Comment (BubentsovKirill) [inline about/index.html:13]: It should be closed.

File: about/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <p><em>day long, and you will feel completely numb and paralyzed – incapable of doing anything else until you drop them.”</em></p> 
+    <br/>
+    <blockquote><strong> Moral of the Story:</strong>
+        <del>On days when you’re stressed out and feeling overwhelmed, learn to embrace the inevitable and let go. 
+        Some things cannot be controlled and no amount of worrying could put all your burdens away. Instead 
+        of letting stress get to you, embrace it and conquer it.</del> Let yesterday’s worries inspire you to a productive day.
+    </blockquote>
+    <hr>
+    <br/>
+    <img style="width:500px"src="https://easternmirrornagaland.com/wp-content/uploads/2020/10/Youre-Breathtaking-Keanu-Reeves.png"alt="Here is Kianu"/>
```
Comment (BubentsovKirill) [inline about/index.html:36]: Please add space before `alt` attribute.

File: index.html (hunk 1 above)
Comment (KsushaDk) [reply index.html]: Put right
<a href="./about/index.html">About author</a>

File: about/index.html (hunk 2 above)
Comment (KsushaDk) [reply about/index.html]: <hr/>

File: about/index.html (hunk 3 above)
Comment (KsushaDk) [reply about/index.html]: <hr/>

**Pull request merged**